dll/win32/shell32/CShellLink.cpp

```diff
@@ -2644,18 +2637,10 @@ HRESULT CShellLink::DoOpen(LPCMINVOKECOMMANDINFO lpici)
     sei.lpVerb = L"open";
 
     // HACK for ShellExecuteExW
```

Comment:
```suggestion
    // HACK for ShellExecuteExW: Change the default verb if this is a Control Panel applet
```

Reply:
See my #7152 that solves this in a different way (and actually works)